client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -101,11 +102,13 @@ export const getTask = (
 		case CHECKLIST_KNOWN_TASKS.BLOGNAME_SET:
 			taskData = {
 				timing: 1,
-				title: translate( 'Give your site a name' ),
-				description: translate(
-					'Give your new site a title to let people know what your site is about.'
-				),
-				actionText: translate( 'Name your site' ),
+				title: isBlogger ? translate( 'Name your blog' ) : translate( 'Give your site a name' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **65** times:
`translate( 'Name Your Blog' )` *ES Score: 8*